javascript/ql/src/Security/CWE-078/SecondOrderCommandInjection.ql

```diff
@@ -0,0 +1,25 @@
+/**
+ * @name Second order command injection
+ * @description Some shell programs allow arbitrary command execution via their command line arguments.
```

Comment:
Could we rewrite this so it follows the [preferred syntax for a query description](https://github.com/github/codeql/blob/main/docs/query-metadata-style-guide.md#query-descriptions-description) which is `"Syntax X causes behavior Y." `?
I think it would be clearer to users if we went straight to the point.

Reply:
I've rewritten it to: `Using user controlled data as arguments to some commands, such as git clone,
can allow arbitrary commands to be executed.`